versions.properties

```diff
@@ -1,12 +1,12 @@
-jacksonVersion=2.8.9
+jacksonVersion=2.8.10
```

Comment:
This may be the best solution we have at the moment. `2.9.8` is the latest version and fixes a number of vulnerabilities, but `api-checker` prevents up from moving past `2.8.10` unless we upgrade Jackson in `api-checker`.

See:
https://github.com/rackerlabs/repose/pull/1849#discussion_r257789007

Reply:
I'm good with only bumping it to what we can for the current `api-checker` and then adding a story to bump the version used in the dependency.

The main goal of adding the OWASP checking was to make us and our customers aware of potential vulnerabilities so we could appropriately guard against them.